lightning-persister/src/lib.rs

```diff
@@ -75,6 +76,12 @@ impl FilesystemPersister {
 		self.path_to_channel_data.clone()
 	}
 
+	pub(crate) fn path_to_monitor_data(&self) -> String {
```

Comment:
Can't you just return the `PathBuf` and use it directly as a `Path` in fs::... calls?